mne_bids/bids_meg.py

```diff
@@ -29,24 +30,26 @@ def _channel_tsv(raw, fname):
 
     map_chs = dict(grad='MEG', mag='MEG', stim='TRIG', eeg='EEG',
                    eog='EOG', misc='MISC')
-    map_desc = dict(grad='sensor gradiometer', mag='magnetometer',
-                    stim='analogue trigger',
-                    eeg='electro-encephalography channel',
-                    eog='electro-oculogram', misc='miscellaneous channel')
+    map_desc = dict(grad='Gradiometer', mag='Magnetometer',
+                    stim='Trigger',
+                    eeg='ElectroEncephaloGram',
+                    ecg='ElectroCardioGram',
+                    eog='ElectrOculogram', misc='Miscellaneous')
 
     status, ch_type, description = list(), list(), list()
     for idx, ch in enumerate(raw.info['ch_names']):
         status.append('bad' if ch in raw.info['bads'] else 'good')
         ch_type.append(map_chs[channel_type(raw.info, idx)])
         description.append(map_desc[channel_type(raw.info, idx)])
-
-    onlinefilter = '%0.2f-%0.2f' % (raw.info['highpass'], raw.info['lowpass'])
-    df = pd.DataFrame({'name': raw.info['ch_names'], 'type': ch_type,
-                       'description': description,
-                       'onlinefilter': onlinefilter,
-                       'samplingrate': '%f' % raw.info['sfreq'],
-                       'status': status
-                       })
+    low_cutoff, high_cutoff = (raw.info['highpass'], raw.info['lowpass'])
+    n_channels = raw.info['nchan']
+    sfreq = raw.info['sfreq']
+    df = pd.DataFrame(OD([('name', raw.info['ch_names']),
+                          ('type', ch_type), ('discription', description),
+                          ('sampling_frequency', ['%.2f' % sfreq] * n_channels),
+                          ('low_cutoff', ['%.2f' % low_cutoff] * n_channels),
+                          ('high_cutoff', ['%.2f' % high_cutoff] * n_channels),
+                          ('status', status)]))
```

Comment:
Sorry, last nitpick. Why not just do:

```py
df = df[['name', 'type', 'sampling_frequency', 'low_cutoff', 'high_cutoff', 'status']]
```

rather than use ordered dict?